rectools/visuals/metrics_app.py

```diff
@@ -160,6 +162,19 @@ def fold_ids(self) -> tp.Optional[tp.List[int]]:
             return sorted(self.data[Columns.Split].unique())
         return None
 
+    @staticmethod
+    def _validate_nbformat_version() -> None:
+        try:
```

Comment:
Instead of checking this ourselves, could we use some `plotly` method to get this error?  
It would be much better, because if requirements changes, say tomorrow they need 4.3.0, it would be handled automatically without any control from our side.

Reply:
Outdated, because we added nbformat in project dependencies